ops/pebble.py

```diff
@@ -2068,23 +2074,27 @@ def _request_raw(
 
         return response
 
+    @tracer.start_as_current_span('ops.pebble.Client.get_system_info')  # type: ignore
```

Comment:
Similar here: could we not trace the lower-level Pebble request to do it only in one place, or is that too hard to make meaningful?

Reply:
Let me think about that.
One issue is being able to provide useful information about the arguments.
For example, what set of services is being start or stopped.